dataService.js

```diff
@@ -0,0 +1,42 @@
+// import required json arrays
+const allCategories = require('./src/data/categories.json').categories
+const allProducts = require('./src/data/products.json').products
+
+/*
+ * Map individual product objects to product id attributes
+ */
+function getProducts (product) {
```

Comment:
This shouldn't take any parameter. You should be creating the whole `productMap` here, using [`allProducts`](https://github.com/riskycase/my-express/pull/3/files#diff-063dfcc1131b3827c39d13e920435d64R3). Basically convert `allProducts` to a map, with `productId` as the key. It's mentioned clearly in step 3.